flow-typed/npm/jest_v24.x.x.js

```diff
@@ -1,7 +1,5 @@
 // flow-typed signature: f5a484315a3dea13d273645306e4076a
 // flow-typed version: 7c5d14b3d4/jest_v23.x.x/flow_>=v0.39.x
-import type {ComponentType} from "react";
```

Comment:
This is to avoid having to move everything into a `module { }` block?

Reply:
Yup.